docs/docs-content/vm-management/vm-migration-assistant/additional-configuration.md

```diff
@@ -0,0 +1,336 @@
+
+### Configure Plan Mappings
+
+The configurable settings are described in the following table. These were originally defined when you performed the
+steps in [Create Migration Plans](./migrate-vms-vmo-cluster.md#create-migration-plan).
+
+| Setting         | Description                                                                                                    | Example                                    |
+| --------------- | -------------------------------------------------------------------------------------------------------------- | ------------------------------------------ |
+| **Network map** | The source provider to target provider network map. Adjust the mapping, or leave the default mapping in place. | `VM-NETWORK` / `Pod Networking`            |
+| **Storage map** | The source provider to target provider storage map. Adjust the mapping, or leave the default mapping in place. | `vsanDatastore1` / `spectro-storage-class` |
+
+Perform the following steps to change a setting.
+
+1. Click on **Edit mappings**.
+2. Click on the **drop-down Menu** for each map and select a different option from the list.
+3. Click **Update mappings** after making changes.
+
+</TabItem>
+
+<TabItem label="Hooks" value="hooks">
+
+The **Hooks** tab displays the migration hooks for the migration plan.
+
+### Configure Plan Hooks
+
+Perform the following steps to enable a hook.
+
+1. Click on **Enable hook** toggle for either **Pre migration hook** or **Post migration hook**.
+
+2. The configurable settings are described in the following table.
+
+   | Setting               | Description                                                                                                                                                                                                                             |
+   | --------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+   | **Hook runner image** | If enabling a pre or post migration hook, provide registry URL to the hook-runner or custom image. Make sure you specify the registry URL without the HTTP scheme `https://` or `http://`. For example, `quay.io/konveyor/hook-runner`. |
+   | **Ansible playbook**  | You can optionally provide an [Ansible playbook](https://ansible.readthedocs.io/projects/runner/en/stable/intro/) for the hook. You can only specify a playbook if you are using a hook-runner image.                                   |
+
+3. Click **Update hooks** after making changes.
+
+</TabItem>
+
+</Tabs>
+
+## Storage Maps
+
+Navigate to your storage maps by clicking **StorageMaps for virtualization** on the left **Main Menu**.
+
+Storage maps are added here after [creating a migration plan](./migrate-vms-vmo-cluster.md#create-migration-plan).
+
+Click on a storage plan name to view the its details.
```

Comment:
The spacing here feels a bit out of place.  We could probably just explain it in a sentence or two and remove the newlines.

Reply:
I've rewritten this to explain what a storage and network map actually is. This should read better now.